l10n_it_withholding_tax_payment/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2015 Alessandro Camilli (<http://www.openforce.it>)
+# Copyright (c) 2019 Matteo Bilotta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Italian Withholding Tax Payment',
+    'version': '12.0.1.0.0',
+    'category': 'Account',
+    'author': 'Openforce, '
+              'Odoo Italia Network, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://odoo-community.org/',
```

Comment:
Dato che "name" non è traducibile in weblate, andrebbe in italiano nella forma indicata nelle linee guida:

https://www.odoo-italia.org/documentazione/sviluppo/guidelines.html#titolo-del-modulo

Se vuoi potrebbe eventualmente essere aggiunto un  "summary", sempre in italiano.

Per uniformità con le attuali convenzioni, il sito web andrebbe indicato con una delle due forme qui sotto, a scelta:
https://github.com/OCA/l10n-italy
https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_withholding_tax_payment

Reply:
Fatto!  
Grazie! 😉